server_versions.json

```diff
@@ -81,50 +77,18 @@
         "version": "1.19.2",
         "link": "https://api.papermc.io/v2/projects/paper/versions/1.19.2/builds/307/downloads/paper-1.19.2-307.jar"
       },
-      {
-        "version": "1.19.1",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.19.1/builds/111/downloads/paper-1.19.1-111.jar"
-      },
-      {
-        "version": "1.19",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.19/builds/69/downloads/paper-1.19-69.jar"
-      },
       {
         "version": "1.18.2",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.18.2/builds/386/downloads/paper-1.18.2-386.jar"
-      },
-      {
-        "version": "1.18.1",
-        "link": "https://papermc.io/api/v2/projects/paper/versions/1.18.1/builds/133/downloads/paper-1.18.1-133.jar"
+        "link": "https://api.papermc.io/v2/projects/paper/versions/1.18.2/builds/388/downloads/paper-1.18.2-388.jar"
       },
       {
         "version": "1.17.1",
         "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.17.1.jar"
       },
-      {
-        "version": "1.17",
-        "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.17.jar"
-      },
       {
         "version": "1.16.5",
         "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.16.5.jar"
       },
-      {
-        "version": "1.16.4",
-        "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.16.4.jar"
-      },
-      {
-        "version": "1.16.3",
-        "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.16.3.jar"
-      },
-      {
-        "version": "1.16.2",
-        "link": "https://dl.thesimplecloud.eu/download/versions/paper/Paper-1.16.2.jar"
-      },
-      {
-        "version": "1.16.1",
```

Comment:
Why did you remove this? I would leave it there

Reply:
1.16.5 is the latest version of the 1.16-Group and has bug fixes, optimizations and balancing,
but this can be leave there, because this is a speedrunner version.